webpack/components/extensions/HostDetails/Tabs/__tests__/packagesTab.test.js

```diff
@@ -117,7 +141,7 @@ test('Can filter by package status', async (done) => {
     getByText,
   } = renderWithRedux(<PackagesTab />, renderOptions());
 
-  // Assert that the errata are now showing on the screen, but wait for them to appear.
+  // Assert that the packages are now showing on the screen, but wait for them to appear.
```

Comment:
In general I think these test comments aren't too useful. We should probably delete them so we stop copy/pasting them everywhere. :smile: